chalice/awsclient.py

```diff
@@ -37,6 +42,28 @@ class ResourceDoesNotExistError(Exception):
     pass
 
 
+class LambdaClientError(Exception):
+    def __init__(self, original_error, context):
+        # type: (Any, LambdaErrorContext) -> None
```

Comment:
I try to avoid `Any` unless there's really not a better way.  Can this not be `Exception` or something more specific?